scripts/dev-bootstrap

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eu
+
+function run() {
+    echo "${@}"
+    "${@}"
```

Comment:
hey dawg i heard you like `make` 😂

Reply:
Ha, did not think of that. I’ve added a comment mentioning it could be turned into a small Makefile.